rust/noosphere-ns/tests/ns_test.rs

```diff
@@ -0,0 +1,110 @@
+#![cfg(not(target_arch = "wasm32"))]
+#![cfg(test)]
+pub mod utils;
+use anyhow::{anyhow, Result};
+use cid::{
+    multihash::{Code, MultihashDigest},
+    Cid,
+};
+use noosphere_core::authority::generate_ed25519_key;
+use noosphere_ns::{NameSystem, NameSystemBuilder};
+
+use utils::{create_bootstrap_nodes, wait_ms};
+
+fn new_cid(bytes: &[u8]) -> Cid {
```

Comment:
We have a slightly better way to do this now: https://github.com/subconsciousnetwork/noosphere/blob/1a0625ad79330d35ca137361297318bdbf29137e/rust/noosphere-storage/src/encoding.rs#L23-L32

```rs
derive_cid::<RawCodec>(&bytes);
```

Also, conventionally we are using `Blake2b256` everywhere. Is `Sha2_256` a hard requirement?

Reply:
Cids are at least currently opaque to the entire name system --  that's changing a bit currently with the Ucan iteration, where Cids can be resolved in a Ucan signature (e.g. accessing a Store), so these tests can safely use `Blake2b256` form, good catch